### DISCUSSION TITLE
AI檢察官

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#167] [General]
**新聞內容:**
「**AI檢察官**」是由中國大陸最大、最繁忙地方檢察機關的上海市浦東新區人民檢察院打造與測試。計畫首席科學家石勇教授表示，這科技能夠減輕檢察官的日常工作量，讓他們專注於更困難的案件。
能在一台桌上型電腦運作，可根據取自人工生成案例描述文本的1000個「表徵」正式起訴每一位嫌犯，目前已能辨認和正式起訴上海最常見的8種罪名，但有人對此有些顧慮，「從技術的角度來看，97%的準確率可能很高，但總是會有可能出錯。當事情發生了，誰將承擔責任？檢察官、機器或演算法的設計者？」
「AI檢察官」只能根據以往經驗提告，無法在不斷變化的社會環境預知輿論對案件的反應。這位檢察官說，「AI可能有助偵測錯誤，但不能取代人類做決定」。
**新聞心得:**
透過這個AI檢察官，以高精準度的辨識度幫助人類檢察官，或許可以高效率地處理分析案件，但是歸咎責任這件事情很難去定義，或許在實行前先說清楚判斷錯誤時造成的成果該由誰負責，在出錯時就能找負責人詢問了。
**新聞來源:**
https://udn.com/news/story/7332/5993141

**Comment by Timmy110403540** [comment]
個人認為AI可以幫忙處理瑣碎小事(耗時的文書工作)等，重要決策還是需由真人檢察官來判定

**Reply by cobb110202008** [reply]
認同，人都有可能無法公正的審判一件案件，往往需仰賴自由心證去判決，AI就更無法根據當下情況做出公正的判決

**Comment by pocper1** [comment]
AI 不能判斷人之常情，可能因為有些感情方面的判決會不太準確

**Reply by kaoyihua27** [reply]
沒錯 人類的世界還是必須帶點感情 那是AI不能取代的

**Comment by minniewei** [comment]
就責任歸屬這個議題，我想達爾文醫療機器也面臨相同的問題

**Comment by briankuo123** [comment]
以我們人類多變且複雜的狀況來說，我想應該很難去證實AI所判斷出來的東西是否正確，我想目前做多應該還是當作輔助來使用。

**Comment by wh20502** [comment]
我覺得可以使用在地方法院之類的，到中央法院之類的才要真人，小案件等等可以使用AI，我覺得法律算是冰冷的，過於注重人情味反而有失法治。

**Comment by jack90325** [comment]
我覺得這種科技主要都只能用在輔助，因為一些案子的實際情況太複雜了，而且有很多隱情

**Comment by 110409538** [comment]
人工智慧的誕生的確可以幫忙處理很多事情。但是，在一些比較需要嚴謹態度的事情上，還是自己處理比較妥當。

**Comment by yasklop** [comment]
感覺AI要當檢察官還是偏難，畢竟有些需要情感介入的事，AI目前可能還是無法做到。

**Comment by Shiba010** [comment]
感覺就模前而言ai檢察官應該還是難以勝過人類。

**Comment by ghost** [comment]
感覺可以判決是由AI和法官一起討論並共同得出一個結果xd

**Comment by Lynn12378** [comment]
AI感覺比較無法用在比較抽象廣泛的法條，如果要使用的話可能只有少部分的法律可以使用，判決還是要依賴人類做出決定

**Comment by Yunhans** [comment]
所以還沒有AI法官

**Comment by LanningYang** [comment]
應該變成一切講求證據，可能會無法達到大多數人預期的結果
